### ISSUE TITLE
Add Active Job/GlobalID serialization support to ActiveNode

### ISSUE THREAD

**Issue by DavidChouinard** [neo4jrb/neo4j#815]
Rails 4.2 introduces Active Job. However, trying to use an ActiveNode instance with Active Job causes a [`ActiveJob::SerializationError`](http://edgeapi.rubyonrails.org/classes/ActiveJob/SerializationError.html) error because Active Job doesn't support GlobalID.

Mixing in `GlobalID::Identification` (i.e. `include GlobalID::Identification`) to an Active Job model should be enough to add support, since GlobalID only requires an `id` and `find(id)` method, which Active Node has. 

I'm not sure why it doesn't work.

**Comment by subvertallchris** [comment]
Good catch, will check it out. Can it wait a few days? A little busy right now but I can give it attention if it's a serious blocker for you.

**Comment by DavidChouinard** [comment]
Yes, it can wait. I was planning to work on a feature I'm building using Active Job sometime early next week.

**Comment by jacobwgillespie** [comment]
I think this is fixed (as in, mixing in `GlobalID::Identification` is working for me).  Perhaps it should automatically be mixed in if it's defined?  I may do a PR if I get adventurous...

**Comment by subvertallchris** [comment]
Hey guys, sorry for just coming back to this now. I think we can include `GlobalID::Identification` automatically if the `neo4j/railtie` is invoked.
